Makefile

```diff
@@ -331,13 +332,21 @@ build-push-package:
 	hack/build-package.sh ${PKG_IMG}:${PKG_IMAGETAG}
 
 .PHONY: build-package
-build-package:
+build-package: \
+	validate-package
 	@chmod 777 ${PWD}/hack/hypershift/package/hcp/addon-operator.yaml
 	$(CONTAINER_ENGINE) run --privileged --rm -v ${PWD}:/workdir quay.io/app-sre/yq:4 -i '.spec.template.spec.containers[0].image = "$(OPERATOR_IMAGE_URI)"' \
 	hack/hypershift/package/hcp/addon-operator.yaml
-	$(CONTAINER_ENGINE) build -t $(PKG_IMG):$(PKG_IMAGETAG) -f $(join $(CURDIR),/hack/hypershift/package/addon-operator-package.Containerfile) . && \
+	$(CONTAINER_ENGINE) build -t $(PKG_IMG):$(PKG_IMAGETAG) -f $(join $(CURDIR),/hack/hypershift/package//addon-operator-package.Containerfile) . && \
 	$(CONTAINER_ENGINE) tag $(PKG_IMG):$(PKG_IMAGETAG) $(PKG_IMG):latest
 
+
+.PHONY: validate-package
+validate-package:
+	@echo "-------- Running package validations --------"
+	$(CONTAINER_ENGINE) run --rm -v $(PWD):/workdir $(PKO_CLI_IMAGE) validate ./workdir/hack/hypershift/package
```

Comment:
`:z` specifies the options for the volume mount. So, when you see `$(CURDIR):/workdir:z` in a Docker/Podman command, it means that you are mounting the current directory on the host to `/workdir` inside the container with the `z` SELinux context option enabled. 

```suggestion
	$(CONTAINER_ENGINE) run --rm -v $(PWD):/workdir:z $(PKO_CLI_IMAGE) validate ./workdir/hack/hypershift/package
```